e2e/portalicious/tests/ViewFilterAndSortRegistrations/FilterRegistrationsByDate.spec.ts

```diff
@@ -47,21 +47,46 @@ test('[34947] Filter registrations by Date selection', async ({ page }) => {
   const registrations = new RegistrationsPage(page);
   const tableComponent = new TableComponent(page);
   // Act & Assert
-  await test.step('Filter registrations columns by date', async () => {
+  await test.step('Filter registration created column by "Equals" date', async () => {
     // Filter Registration column by date
     await tableComponent.filterColumnByDate({
       columnName: 'Registration created',
       day,
+      range: 'Equals',
     });
     await tableComponent.validateAllRecordsCount(4);
+    await tableComponent.clearAllFilters();
     // Filter by different date
     const diffDay = currentDate.getDate() === 1 ? 2 : 1;
     if (diffDay !== undefined) {
       await tableComponent.filterColumnByDate({
         columnName: 'Registration created',
         day: diffDay,
+        range: 'Equals',
       });
     }
     await registrations.validateRegistrationIsNotPresent();
+    await tableComponent.clearAllFilters();
+  });
+
+  await test.step('Filter registration created column by "Less than" date', async () => {
+    // Filter Registration column by date
+    await tableComponent.filterColumnByDate({
+      columnName: 'Registration created',
+      day,
+      range: 'Less than',
+    });
+    await registrations.validateRegistrationIsNotPresent();
+    await tableComponent.clearAllFilters();
+  });
+
+  await test.step('Filter registration created column by "Greater than" date', async () => {
+    // Filter Registration column by date
+    await tableComponent.filterColumnByDate({
+      columnName: 'Registration created',
+      day,
+      range: 'Greater than',
+    });
+    await registrations.validateRegistrationIsNotPresent();
```

Comment:
`validateRegistrationIsNotPresent` actually check whether the string `No results found` is found on the page. In our frontend code this is the default behaviour of any empty query table. So I was thinking it may make more sense to move this function to the `TableComponent` file. 

Also maybe we should have a different name for that function? Maybe assertEmptyTableState() or validateEmptyTableMessage()